shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -270,6 +272,16 @@ protected ScanInterpreter getInterpreter(final CommandLine cl, final String tabl
 
   protected void fetchColumns(final CommandLine cl, final ScannerBase scanner,
       final ScanInterpreter formatter) throws UnsupportedEncodingException {
+
+    if ((cl.hasOption(scanOptCfOptions.getOpt()) || cl.hasOption(scanOptColQualifier.getOpt()))
+        && cl.hasOption(scanOptColumns.getOpt())) {
+
+      String formattedString =
+          String.format("Options - %s AND (- %s" + "OR - %s are mutually exclusive ",
```

Comment:
The spacing is a little off here. This is how it looks in the shell: `Options - c AND (- cfOR - cq are mutually exclusive`